File: 백준/Bronze/10798. 세로읽기/세로읽기.cc
#include<iostream>
#include<string>
#include<algorithm>

using namespace std;

int main() {


	string s[5];
	int maxLen = 0;

	for (int i = 0; i < 5; i++) {
		cin >> s[i];
		maxLen = max(maxLen, int(s[i].length()));
	}

	for (int i = 0; i < maxLen; i++) {
		for (int j = 0; j < 5; j++) {
			if (s[j].length() <= i) continue;
			cout << s[j][i];
		}
	}


	return 0;
}

File: 백준/Bronze/10798. 세로읽기/세로읽기.py
S = [list(input().split()) for i in range(5)]

MaxLen = 0

for i in range(5):
    MaxLen = max(MaxLen, len(S[i][0]))

# print(MaxLen)

for i in range(MaxLen):
    for j in range(5):
        if len(S[j][0]) > i:
            print(S[j][0][i], end="")